packages/oc-s3-storage-adapter/package.json

```diff
@@ -14,7 +14,8 @@
   "keywords": [
     "oc",
     "opencomponents",
-    "s3"
+    "goolge storage",
+    "gs"
```

Comment:
ah see previous comment, I guess for some reason those are the changes you wanted to be in the gs, not on the s3 package, probably something went wrong when checking in the files